app/src/androidTest/java/org/kiwix/kiwixmobile/language/LanguageFragmentTest.java

```diff
@@ -43,7 +43,6 @@
 import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
 import static androidx.test.espresso.matcher.ViewMatchers.withContentDescription;
 import static androidx.test.espresso.matcher.ViewMatchers.withId;
-import static androidx.test.espresso.matcher.ViewMatchers.withText;
```

Comment:
Why is it necessary here?

Reply:
the import was unused, should i add it again?

Reply:
Yes! It's not related to PR.